README.md

```diff
@@ -354,6 +354,15 @@ model_from_disk = ViTExtractor(weights=oml.const.CKPT_SAVE_ROOT / "vits16_cars.c
 ```
 [comment]:checkpoint-end
 
+We also have some pretrained ViT-CLIP models with their zeros-shot scores (cmc@1) presented below:
```

Comment:
this is a table not only for clip, so, let's say `pretrained model` instead

Reply:
`zeros-shot` may be a confusing term. I suggest removing it:
`with their values of CMC@1 presented below:`

Reply:
Actually, I'm not so sure about removing this because we are basically saying the same thing about models we have trained on these datasets. So if we leave only "pretrained ViT-CLIP models" in the sentence without explicitly stating that this is zero-shot performance, it may be confusing.
What do you think?

Reply:
Maybe we can say "CLIP-pretrained ViT models" 🤔